libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.m

```diff
@@ -169,4 +196,21 @@ - (DDLogLevel)readLogLevel {
     return logLevel;
 }
 
+static inline DDLogFlag DDLogFlagForLogLevel(DDLogLevel level) {
+    switch (level) {
+        case DDLogLevelOff:
```

Comment:
Shouldn't it be 0 for off?

Looking [here](http://cocoadocs.org/docsets/CocoaLumberjack/2.0.0/Constants/DDLogLevel.html), the level is the bit mask and flags are individual bits. What's the meaning of that function then?